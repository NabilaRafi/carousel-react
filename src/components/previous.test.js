import React from 'react';
import { create } from 'react-test-renderer';
import Previous from './previous';

const suite = 'Testing Previous component <Previous />';

describe(suite, () => {
    const component = create(<Previous />).toJSON();
    test('renders previous component without breaking', () => {
    expect(component).toMatchSnapshot();
    });
});