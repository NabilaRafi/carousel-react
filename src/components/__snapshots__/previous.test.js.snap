// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Previous component <Previous /> renders previous component without breaking 1`] = `
<div
  className="prev-arrow"
>
  <img
    alt="previous"
    src="arrow.svg"
  />
</div>
`;
